service-jehlomat/src/main/kotlin/services/Mailer.kt

```diff
@@ -43,7 +43,7 @@ class FakeMailer: MailerService {
 
 
 class Mailer: MailerService {
-    private val publicUrl = System.getenv("JWT_ISSUER")
+    private val publicUrl = System.getenv("FRONTEND_URL")
```

Comment:
To je nová proměnná, musí se propsat ze secrets, skrz AWS až do dockeru. Je potřeba upravit soubory 
- .github/workflows/terraform_apply.yml
- infrastructure/aws/ecs/service-jehlomat.tmpl
- infrastructure/aws/development.tf
- infrastructure/aws/variables.tf